namemysprint.py

```diff
@@ -37,18 +38,39 @@
               'youthful', 'zealous', 'zen']
 
 # Cyclists, of course.
-NAMES = ['Armstrong', 'Basso', 'Boonen', 'Cancellara', 'Cavendish',
-         'Contador', 'Danielson', 'Evans', 'Froome', 'Hincapie', 'Horner',
-         'Hoy', 'Hushovd', 'Julich', 'Kloden', 'Landis', 'Leipheimer',
-         'Merckx', 'Museeuw', "O'Grady", 'Phinney', 'Powers', 'Renshaw',
-         'Schleck', 'Thomas', 'Ullrich', 'Uran', 'Valverde', 'VanGarderen',
-         'Vaughters', 'Voigt', 'Vos', 'Wiggins', 'Zabel', 'Zabriskie']
+CYCLISTS = ['Armstrong', 'Basso', 'Boonen', 'Cancellara', 'Cavendish',
+            'Contador', 'Danielson', 'Evans', 'Froome', 'Hincapie', 'Horner',
+            'Hoy', 'Hushovd', 'Julich', 'Kloden', 'Landis', 'Leipheimer',
+            'Merckx', 'Museeuw', "O'Grady", 'Phinney', 'Powers', 'Renshaw',
+            'Schleck', 'Thomas', 'Ullrich', 'Uran', 'Valverde', 'VanGarderen',
+            'Vaughters', 'Voigt', 'Vos', 'Wiggins', 'Zabel', 'Zabriskie']
 
+STARWARS = ['Ackbar', 'Amidala', 'Anakin', 'Andor', 'BB-8', 'Binks', 'C-3PO',
+            'Calrissian', 'Chewbacca', 'Cody', 'Dameron', 'Darth Vader',
+            'Darth Tyranus', 'Darth Maul', 'Darth Sidius', 'Dooku', 'Erso',
+            'Fett', 'Finn', 'Grievous', 'Jabba', 'Qui-Gon Jinn', 'Luke',
+            'Maz Kanata', 'Kenobi', 'Organa', 'Leia', 'Palpatine', 'Phasma',
+            'R2-D2', 'Kylo Ren', 'Shmi', 'Skywalker', 'Snoke', 'Solo', 'Windu',
+            'Yoda', ]
+
+
+NAME_PACKS = {'cyclists': CYCLISTS, 'starwars': STARWARS}
```

Comment:
Instead of this dictionary, just capitalize the argparse argument given when pulling from the list.